app/grandchallenge/algorithms/templates/algorithms/job_status_badge_detail.html

```diff
@@ -0,0 +1,7 @@
+<span class="badge badge-{{ object.status_context }}" {% if not object.finished %}hx-get="{% url 'algorithms:job-status-badge-detail' slug=object.algorithm_image.algorithm.slug pk=object.pk %}" hx-trigger="load delay:30s" hx-swap="outerHTML"{% endif %}>
```

Comment:
Due to using subdomains we had to re-implement the url tag. We should discuss that part of the code base in our next 1-1 as it affects a lot of things.

```suggestion
{% load url %}

<span class="badge badge-{{ object.status_context }}" {% if not object.finished %}hx-get="{% url 'algorithms:job-status-badge-detail' slug=object.algorithm_image.algorithm.slug pk=object.pk %}" hx-trigger="load delay:30s" hx-swap="outerHTML"{% endif %}>
```